script.js

```diff
@@ -21,4 +21,93 @@ function closeMenu() {
     logo.classList.remove('active');
     nav.classList.remove('active');
 
+}
+
+// pop-up window
+
+const popupHeading = ["Multi Post Stories", "Keeping track of hundreds  of components website"];
+const popImg = ["./images/popupMobile.svg", "./images/live.svg", "./images/GitHub.svg"];
+const popupDescription = [" Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita."];
+const popupList = ["html", "Bootstrap", "Ruby on rails"] ;
+const popupLinks = ["https://the-normal-one1.github.io/PortFolio/", "https://github.com/The-Normal-One1/PortFolio"]
```

Comment:
- [ ] I must commend your approach to displaying the project data dynamically 👍🏽 . I think it will be better if you add all your project data in a single array so that it will be easier to loop through the project data instead of repeating yourself 😄 

e.g

```
const projectDatas = [
   {
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
},
{
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
},
{
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
};
]
```